recipe/meta.yaml

```diff
@@ -24,6 +27,7 @@ requirements:
     - python
     - pip
     - setuptools
+    - wheel
     - numpy
   run:
     - python
```

Comment:
Seems like coloram is not required and that packaging >20.9 is required.
https://github.com/slundberg/shap/blob/v0.41.0/setup.py#L229

Reply:
Re: `packaging`, I thought that was required primarily for installation, and thus we would be exempt from that requirement. let me confirm that. Thanks for catching colorama

Reply:
You are right, `packaging` is a runtime requirement after all. Thank you!